helm/sematic-server/templates/configmap.yaml

```diff
@@ -14,3 +14,4 @@ data:
   AWS_S3_BUCKET: {{ .Values.aws.storage_bucket | quote }}
 {{ end }}
   KUBERNETES_NAMESPACE: {{ .Release.Namespace }}
+  SEMATIC_WORKER_KUBERNETES_SA: {{ .Values.app_config.worker_kubernetes_sa | default "default" | quote }}
```

Comment:
i'd remove the `| default "default"` from here, since we're already providing a default in our `Values.yml`, and it would be best to have only one location for the default specification.

Reply:
The idea was that if somebody is using their own `values.yaml`, it would be backwards-incompatible to add this without setting a default. However, considering that nobody is actually using this chart yet, I suppose it doesn't matter.